packages/astro/src/content/content-layer.ts

```diff
@@ -114,21 +116,19 @@ class ContentLayer {
 	async #getLoaderContext({
 		collectionName,
 		loaderName = 'content',
-		parseData,
 		refreshContextData,
 	}: {
 		collectionName: string;
 		loaderName: string;
-		parseData: LoaderContext['parseData'];
 		refreshContextData?: Record<string, unknown>;
-	}): Promise<LoaderContext> {
+	}): Promise<Omit<LoaderContext, 'store'>> {
 		return {
 			collection: collectionName,
-			store: this.#store.scopedStore(collectionName),
 			meta: this.#store.metaStore(collectionName),
 			logger: this.#logger.forkIntegrationLogger(loaderName),
 			config: this.#settings.config,
-			parseData,
+			// TODO: remove in v7
+			parseData: async (props) => props.data,
```

Comment:
I may remove it in v6 after all since v5 loaders probably won't work in v6 anyways

Reply:
They'll all break?